Dockerfile.prod

```diff
@@ -0,0 +1,34 @@
+FROM python:3.10
+
+ENV PYTHONUNBUFFERED 1
+ARG REQUEST_BROKER_DNS
+ARG APPLICATION_PORT
+RUN apt-get update
+RUN apt-get install --yes apache2 apache2-dev 
+RUN apt-get install --yes postgresql
```

Comment:
I'm seeing postgres being installed in the current Dockerfile but I think that's a mistake - I don't think this is needed.